Model/Api/OrderManagement.php

```diff
@@ -179,10 +179,17 @@ public function manage(
             $this->bugsnag->notifyException($e);
             $this->metricsClient->processMetric("webhooks.failure", 1, "webhooks.latency", $startTime);
             $this->response->setHttpResponseCode(422);
+            $error_code = $e->getCode();
+            if (($error_code != 6301 && $error_code != 6303)) {
```

Comment:
We should have error codes defined as constants.
M1 reference https://github.com/BoltApp/bolt-magento1/blob/master/app/code/community/Bolt/Boltpay/Model/Productpage/Cart.php

Reply:
Done